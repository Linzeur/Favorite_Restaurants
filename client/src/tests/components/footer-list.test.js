import React from "react";
import { render } from "@testing-library/react";

import Footer from "../../components/footer-list";

test("footer-list component", () => {
  const { asFragment } = render(
    <Footer nroResult={5} nameElem={"locations"} />
  );

  expect(asFragment()).toMatchSnapshot();
});
