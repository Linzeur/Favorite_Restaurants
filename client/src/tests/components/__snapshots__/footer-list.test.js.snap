// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer-list component 1`] = `
<DocumentFragment>
  <div
    class="css-gn3t32-Footer"
  >
    <span
      class="css-8ep07x-Footer"
    >
      Results:
    </span>
    <span
      class="css-xp9py7-Footer"
    >
      5 locations
    </span>
  </div>
</DocumentFragment>
`;
